SampleUsages/TestScenario.cs

```diff
@@ -26,7 +27,7 @@ class TestScenario
         public int WarmUpTimeInMinutes { get; set; }
         private ILogger _logger { get; set; }
 
-        internal void RunScenario(ILogger logger)
+        internal void RunScenario(ILogger logger, ServerlessResultManager.TestScenario testScenario = null)
```

Comment:
This method is a bit confusing - the name makes it sound like you are running a scenario, but the scenario is actually optional. Looking at the code it appears we only use the scenario ID. Would it make more sense for this to be called RunTest and for it to take an optional scenario ID?